src/main/java/se/kits/gakusei/content/model/Lesson.java

```diff
@@ -99,6 +99,15 @@
             inverseJoinColumns = @JoinColumn(name = "nugget_id", referencedColumnName = "id"))
     private List<Nugget> nuggets;
 
+    @ManyToMany
+    @JoinTable(
+            name = "lessons_kanjis",
+            schema = "contentschema",
+            joinColumns = @JoinColumn(name = "lesson_id", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "kanji_id", referencedColumnName = "id")
```

Comment:
kan vara bra att lägga in uniqueConstraint här så vi slipper duplicates
uniqueConstraints = @UniqueConstraint(columnNames = {"lesson_id", "kanji_id"})

Reply:
🏁